NOTICE.txt

```diff
@@ -5629,8 +5629,7 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 --------------------------------------------------------------------
 Dependency: google.golang.org/api
-Version: v0.7.0
```

Comment:
:thinking: any idea why this version is being removed? it still appears in the `vendor.json`.